tests/integration/test_certs/client_1024_rsa.cert

```diff
@@ -0,0 +1,19 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I'm assuming this is signed using `RSA-SHA256`. Should we add integration test for every (sig, hash) that we claim to support? I don't think we have test coverage on the "non-standard" sigalgs. I think we can just issue a client cert for each pair(RSA-SHA1, RSA-SHA224, RSA-SHA256, RSA-SHA384, RSA-SHA512) and iterate over them in the integration test.